packages/turf-voronoi/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@turf/voronoi",
+  "version": "0.1.0",
```

Comment:
The version can be `5.0.0` since this will be published under that major release.